moveit_ros/moveit_servo/src/servo.cpp

```diff
@@ -367,32 +350,53 @@ Eigen::VectorXd Servo::jointDeltaFromCommand(const ServoInput& command, const mo
     }
     else if (expected_type == CommandType::TWIST)
     {
-      try
+      const auto planning_frame_maybe = getIKSolverBaseFrame(robot_state, active_subgroup_name);
+      if (planning_frame_maybe.has_value())
       {
-        const TwistCommand command_in_planning_frame = toPlanningFrame(std::get<TwistCommand>(command));
-        delta_result =
-            jointDeltaFromTwist(command_in_planning_frame, robot_state, servo_params_, joint_name_group_index_map);
-        servo_status_ = delta_result.first;
+        const auto& planning_frame = *planning_frame_maybe;
```

Comment:
You can, but this is standard to make it less awkward in follow-on calls. And because it's being extracted by reference, it's a pretty inexpensive operation.